### PR TITLE
Milestone#2(Add and Remove)

### PR DESCRIPTION
**Pull request by Dinma-Faith-zz**
**This is the second milestone of my to-doList**

- I created another js file and added functions to add and remove task-list
- This project is built with a simple HTML structure, CSS, and JS.

Feel free to drop your comment and suggestions. I am very open to improvement
Thanks

### CODE REVIEW

File: src/list.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  static delTask(index, taskList) {
+    taskList = taskList.filter((tasks, i) => index !== i);
+    localStorage.setItem('taskDetail', JSON.stringify(taskList));
+    window.location.reload();
+    taskList.forEach((taskindex, i) => {
+      taskindex.index = i;
+      localStorage.setItem('taskDetail', JSON.stringify(taskList));
+    });
+  }
```
Comment (phelian23) [inline src/list.js]: - [ ] Good job implementing functions to add and delete todos, kindly add a function to implement the editing functionality. Make sure that after editing, the todo task is saved, and its persistent on reload.

File: src/list.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    taskList.push(myTask);
+    localStorage.setItem('taskDetail', JSON.stringify(taskList));
+  }
+
+  static delTask(index, taskList) {
+    taskList = taskList.filter((tasks, i) => index !== i);
+    localStorage.setItem('taskDetail', JSON.stringify(taskList));
+    window.location.reload();
+    taskList.forEach((taskindex, i) => {
+      taskindex.index = i;
```
Comment (phelian23) [inline src/list.js]: - [ ] It is a requirement that the index property is set to the value of the new array length i.e the first task has an index equal to 1, 2nd task with an index value of 2. Presently the task index value is taking the value of the default index i.e first task has an index equal to zero. Kindly fix this.

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @Dinma-Faith, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ Linters are passing
✅ Descriptive pull request
✅ Good commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by arslanbisharat** [comment]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**